### DISCUSSION TITLE
Some Tentative Guiding Goals for IPUMS.jl

### DISCUSSION THREAD

**Discussion by TheCedarPrince** [JuliaHealth/IPUMS.jl#4] [Ideas]
Following some discussion with @00krishna, here are what I think the tentative goals of this package should be at the moment and to help with guiding development:

- [ ] Draft some core functionalities of IPUMS.jl based on ipumsr
- [ ] Submit a small abstract to JOSS on IPUMS.jl
- [ ] Explore how IPUMS.jl can compose with the rest of the ecosystem
- [ ] Prototype a study from a previous ipumsr based paper (of a kind)
- [ ] Draft a small white paper or conference submission on this

Exploration will be quite open-ended but then expanding on a previous study may also be fun.

**Comment by 00krishna** [comment]
This is a good list. One thing that might also be really good, is to target the development of some youtube videos for the usage of our julia package. There is an IPUMs youtube channel that has a few subscribers. But it is a new audience and I imagine lots of new gradate students who are getting into IPUMs data might look at the channel. So we can think about that. We can then use the same video or slides for a JuliaCon presentation, etc. 

Here is the channel. 

https://www.youtube.com/@MPCIPUMS
